rowan/project.ston

```diff
@@ -0,0 +1,11 @@
+RwProjectSpecificationV2 {
+  #specName : 'project',
+  #projectSpecPath : 'rowan',
+  #componentsPath : 'rowan/components',
+  #packagesPath : 'source',
+  #projectsPath : 'rowan/projects',
+  #specsPath : 'rowan/specs',
+  #packageFormat : 'tonel',
+  #packageConvention : 'RowanHybrid',
```

Comment:
```suggestion
  #packageConvention : 'Rowan',
```

Reply:
Tried this initially but Rowan would complain 'Disk package convention ''RowanHybrid'' does not match expected package convention ''Rowan'''

Reply:
Is it consistent with the one declared in `source/.properties` ?

Reply:
That was it, thanks. dba062ba4e7eb75b00a993a26a2eb8b077118d3e